scripts/eth3d_benchmark.sh

```diff
@@ -0,0 +1,113 @@
+# Script to launch jobs over various datasets & front-ends.
```

Comment:
this script looks good. I might mention ETH3D explicitly in the comment here, e.g.
```
# Script to launch jobs over various ETH3D datasets & front-ends.
# See https://www.eth3d.net/data/schoeps2017cvpr.pdf for more details.
```

Reply:
added

Reply:
Marking as resolved, thanks.